index.html

```diff
@@ -38,8 +41,190 @@ <h1>I'm Mucha <br> Glad to see <br> you!</h1>
+            <img src="images/Snapshoot-Portfolio-4.png" alt="card-4-img">
+        </div>
+
+        <div class="work-content">
+
+
+            <h2 class="tonic">Multi-Post<br>Stories</h2>
+
+            <ul class="prop">
+                <li class="can">CANOPY</li>
+                <li class="bac"><img src="images/Counter.png" alt="dot"> Back End Dev</li>
+                <li class="num"><img src="images/Counter.png" alt="dot"> 2015</li>
+            </ul>
+
+            <p class="tonic-def">A daily selection of privately<br> personalized reads; no accounts or<br> sign-ups
+                required.</p>
+
+            <ul class="stack">
+                <li class="html">html</li>
+                <li class="css">css</li>
+                <li class="js">javaScript</li>
+            </ul>
+
+            <div class="button">
+                <button type="button">See Project</button>
+            </div>
+        </div>
+    </section>
+
+    <!-- about me  -->
+    <section class="introd">
+        <h2>About<br> Myself</h2>
+        <p>I'm a software developer! I can help you <br> build a product, feature or website Look <br> through some
+            of my work and <br> experience! If you like what you see and <br> have a project you need coded, don't
+            <br> hestiate to contact me
+        </p>
+
+        <a href="#" class="con">LET'S CONNECT</a>
+
+        <ul class="socials">
+            <li><a href="#"><img src="images/github.svg" alt="Github"></li>
+            <li><a href="#"><img src="images/Linkedin icon.svg" alt="linkedin"></a></li>
+            <li><a href="#"><img src="images/angellist.svg" alt="angellist"></a></li>
+            <li><a href="#"><img src="images/twitter.svg" alt="twitter"></a></li>
+            <li><a href="#"><img src="images/medium.svg" alt="medium"> </a></li>
+        </ul>
+
+        <div class="button">
+            <button type="button">Get my resume</button>
+        </div>
+
+        <div class="lang">
+            <ul class="lang-head">
+                <li><h3>Languages</h3></li>
+                <li><img src="images/chev-down.png" alt=""></li>
+            </ul>
+
+            <ul class="lang-list">
+                <li class="script"><img src="images/Ellipse 1.png" alt="js"> JavaScript</li>
+                <li class="htm"><img src="images/Ellipse-html.png" alt="html"> HTML</li>
+                <li class="cs"><img src="images/Ellipse-css.png" alt="css"> CSS</li>
+            </ul>
+
+
+            <ul class="frame">
+                <li><h3>Frameworks</h3></li>
+                <li><img src="images/Icon - Chevron - Right - Larger.png" alt=""></li>
+            </ul>
+
+            <img src="images/Separator Bottom.png" alt="">
+
+            <ul class="skl">
+                <li><h3>Skills</h3></li>
+                <li><img src="images/Icon - Chevron - Right - Larger.png" alt=""></li>
+            </ul>
+
+            <img src="images/Separator Bottom.png" alt="">
+
```

Comment:
- [ ] As requested by the previous reviewer, kindly adopt the best practice by adding `alt text` to every image tag you have on your page. Apart from being semantically correct, it also helps the screen readers, visually impaired people to know where the images are on your page. Kindly them accordingly.